prefeRx/src/test/java/com/vinted/preferx/EnumPreferenceImplTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.vinted.preferx
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.test.core.app.ApplicationProvider
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class EnumPreferenceImplTest {
+
+
+    private val appContext = ApplicationProvider.getApplicationContext<Context>()
+    private val sharedPref = appContext.getSharedPreferences("TestPrefs", Context.MODE_PRIVATE)
+    private val sharedPrefEditor: SharedPreferences.Editor = sharedPref.edit()
+    private val fixture = EnumPreferenceImpl(sharedPref, "key", TestEnum.BAR)
+
+    @Test
+    fun beforeEach() {
+        sharedPrefEditor.clear()
+    }
```

Comment:
The name here implies the function is executed before each test, but is annotated with `@Test`